include/aws/cryptosdk/private/framefmt.h

```diff
@@ -33,6 +34,37 @@ struct aws_cryptosdk_frame {
     struct aws_byte_buf authtag;
 };
 
+// MAX_FRAME_SIZE = 2^32 - 1
```

Comment:
I assume these magic numbers from from the spec. If so, there should be a reference to the part of the spec doc that defines this.

Reply:
That makes sense. However, I don't have a link to the spec. Could someone point to it?